<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookDetail</class>
 <widget class="QMainWindow" name="BookDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="fieldLanguage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelISBN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAuthor">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPublishedOn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Published On:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldISBN">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPublishedOn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldSize">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>340</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCEL / RETURN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActionStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>345</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+ Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPublishedAt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mainBookAddLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Adelle&quot;;</string>
    </property>
    <property name="text">
     <string>Book Manager ~ Detail Book</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPageNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Page Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLanguage">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Language:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPublishedAt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Published At:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldPagesNumber">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldAuthor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fieldDescription">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>571</width>
      <height>79</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMy_Account">
    <property name="title">
     <string>My Account</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuMy_Account"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
